### PR TITLE
1188 second iteration

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Added non processed asset group sightings to the output 
- Filtered out internal users using bitwise & operation in the query

### CODE REVIEW

File: app/modules/site_settings/resources.py [hunk 1]
```diff
@@ -535,7 +535,20 @@ def get(self):
         if is_module_enabled('users'):
             from app.modules.users.models import User
 
-            response['num_users'] = User.query_search().count()
+            num_internal_users = User.query.filter(
+                User.static_roles.op('&')(User.StaticRoles.INTERNAL.mask) > 1
```
Comment (hwindsor) [inline app/modules/site_settings/resources.py:539]: Something I have bee wondering for a while is how to do an sqlachemy query filtering on user roles.
bitwise & against the mask actually worked. 
The comparison could have been against the mask but anythning that is non 0 matches.

File: tasks/app/site_settings.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def get_value(context, key, default=None):
 
 
 @app_context_task()
-def get_public_data(context):
+def get_public_data(context, debug=False):
+
+    if debug:
```
Comment (drewblount) [inline tasks/app/site_settings.py:50]: do we want to push this debug option to main?

**Review by drewblount — APPROVED** [review]
LGTM

**Review by naknomum — APPROVED** [review]
looks good

File: tasks/app/site_settings.py (hunk 2 above)
Comment (hwindsor) [reply tasks/app/site_settings.py:50]: Absolutely.
In a pyinvoke task this is extremely useful.
It allows a user (who is logged into the server) to be able to  really interrogate the system to diagnose what is actually happening.

**Pull request merged**